cfgov/unprocessed/apps/prepaid-agreements/css/main.less

```diff
@@ -0,0 +1,119 @@
+@import (reference) '../../../css/main.less';
+@import (less) '../../../css/search.less';
+@import (less) './search-bar.less';
+
+.u-readable-width {
+    max-width: 41.875rem;
+}
+
+.u-break-word {
+    word-break: break-word;
+}
+
+.prepaid-agreements {
+    dt, dd {
+        display: block;
+        margin-left: 0;
+    }
+
+    dt {
+        letter-spacing: 0;
+        font-weight: normal;
+        text-transform: none;
+        font-size: inherit;
+        .heading-4;
```

Comment:
Let's do `.heading-4();` to better show this is a mixin.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96